holmes/core/conversations.py

```diff
@@ -249,6 +255,9 @@ def build_issue_chat_messages(issue_chat_request: IssueChatRequest, ai: ToolCall
             },
         ]
 
+    if global_instructions and global_instructions.instructions and len(global_instructions.instructions[0]) > 0:
```

Comment:
Im not 100% precent sure since not familiar enough with the code but this flow is part of the conversation history right? So it mean that the history should already include the globa instructions no? why we are resending this?

Reply:
We have global instructions saved in the database and fetched every time because user might want to change it dynamically during the chats, and continue to chat after. The most effective method is to pass them through user_prompt every time, and define rules for using them in system_prompt

Reply:
Not sure I understand, why we move them in some cases as user prompt and in other cases as system prompt? why it isn't always system prompt?